js/index.js

```diff
@@ -1 +1,102 @@
 // Your code goes here
+
+const myBtn = document.querySelector('.content-pick .btn');
+const contentPick = document.querySelector('.content-pick');
+const myHeader = document.querySelectorAll('.home h2');
+const myNav = document.querySelector('body');
+const readerProgress = document.querySelector('.progress');
+const introImg = document.querySelector('.intro img');
+const imgScooby = document.querySelector('.content-section img');
+const imgWendy = document.querySelector('.inverse-content img');
+const navPrevent = document.querySelectorAll('.nav-container a');
+const logoText = document.querySelector('.nav-container .logo-heading');
+
+
+console.log(logoText);
+
+window.addEventListener('load', () => {
```

Comment:
Very cool use of an event listener.  I like how you have changed all of the images when hovering.